.github/workflows/CI.yml

```diff
@@ -79,6 +85,187 @@ jobs:
           which pip
           python -m pip install -U pip
           python -m pip install -U --prefix "${INSTALLDIR}/conda" -r requirements-dev.txt
+          cd "${HOME}"
+          zip -vr taucmdr.zip ${INSTALLDIR##*/} -x "taucmdr/system/src/*" -x "taucmdr/conda/*"
+
+      - name: Upload TAU Commander install log
+        if: always()
+        uses: actions/upload-artifact@v1
+        with:
+          name: taucmdr-tau-minimal-debug-log-${{ matrix.os }}
+          path: ${{ steps.info.outputs.home_dir }}/.local/taucmdr/debug_log
+
+      - name: Upload TAU Commander minimal install
+        if: always()
+        uses: actions/upload-artifact@v1
+        with:
+          name: taucmdr-installation-${{ matrix.os }}
+          path: ${{ steps.info.outputs.home_dir }}/taucmdr.zip
+
+  Install-taucmdr-full:
+    needs: Install-taucmdr-minimal
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: true
+      matrix:
+        os: [ubuntu-18.04, ubuntu-16.04]
+        python_version: ['2.7']
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 9999
+
+      - name: Install system packages
+        run: |
+          sudo apt-get update -qq || true
+          sudo apt-get install -y --no-install-recommends gcc g++ gfortran mpi-default-bin mpi-default-dev
+
+      # - name: Setup Python
+      #   uses: actions/setup-python@v1
+      #   with:
+      #     python-version: '3.6'
+      #     architecture: 'x64'
+
+      - name: System info
+        id: info
+        run: |
+          uname -a
+          mount
+          gcc --version
+          echo "github.event_name: ${{ github.event_name }}"
+          echo "github.ref:        ${{ github.ref }}"
+          echo "matrix.os:         ${{ matrix.os }}"
+          export INSTALLDIR="${HOME}/${INSTALLDIR}"
+          echo "::set-output name=install_dir::${INSTALLDIR}"
+          echo "::set-env name=INSTALLDIR::${INSTALLDIR}"
+          echo "::set-output name=home_dir::${HOME}"
+          echo "::set-output name=taucmdr_system::${INSTALLDIR}/${__TAUCMDR_SYSTEM_PREFIX__}"
+          echo "::set-env name=__TAUCMDR_SYSTEM_PREFIX__::${INSTALLDIR}/${__TAUCMDR_SYSTEM_PREFIX__}"
+          echo "::add-path::${INSTALLDIR}/conda/bin"
+          export PATH="${INSTALLDIR}/conda/bin:${PATH}"
```

Comment:
YAML anchors and aliases would allow this code not to be duplicated, but until GitHub actions supports this, we'll have to make do with something like what we currently have.